docs/remote/containers.md

```diff
@@ -1163,11 +1165,19 @@ See [Docker Desktop for Windows tips](/docs/remote/troubleshooting.md#docker-des
 
 ### Can I use Podman instead of Docker?
 
-Podman 1.9+ is mostly compatible with Docker's CLI commands and therefore generally does work if you update the setting **Remote > Containers: Docker Path** to `podman` on Linux.
+Podman 1.9+ is mostly compatible with Docker's CLI commands and therefore generally does work if you update the setting **Remote > Containers: Docker Path** to `podman` on Linux. 
 
 ![Docker Path setting](images/containers/docker-path-setting.png)
 
-However, certain tricks like [Docker-from-Docker do not work](https://github.com/containers/libpod/issues/4056#issuecomment-535511841) due to limitations in Podman. This affects the **Remote-Containers: Try a Sample...** and **[Remote- Containers: Open repository in container...](#quick-start-open-a-git-repository-or-github-pr-in-an-isolated-container-volume)** commands.
+You [may need to symbolically link](https://github.com/microsoft/vscode-remote-release/issues/116#issuecomment-619329668[Podman support · Issue #116 · microsoft/vscode-remote-release · GitHub](https://github.com/microsoft/vscode-remote-release/issues/116#issuecomment-619329668)) docker to podman in order for the extension to function when docker and docker-compose are not installed on the system. If not linked, the extension will require docker to be installed even if you change the executable docker path settings of the extension mentioned above.
```

Comment:
With the above setting the symbolic link is no longer needed.

Reply:
@chrmarti yesterday I went through and installed the supported .rpm based installation of VSCode and the latest extension version and I still needed to create a soft link in order for it not to ask for docker to be installed. I'll add additional notes in the appropriate linked issue to provide more insight on that case, which is why I included this section with a link to that issue. 

This issue may also affect the flathub/flatpak repackages of VSCode for a while-- so even if its patched with the latest version of the extension and IDE, the behavior may still persist for some time until the unsupported community distributions catch up.